### ISSUE TITLE
[Bug]: Container Labels reset to default after editing

### ISSUE THREAD

**Issue by spectaA** [coollabsio/coolify#4498]
### Error Message and Logs

TL;DR
It seems that it's not possible to customize Container Labels anymore.

**[Previous versions behaviour]**

I used to deploy a multi-tenant app and update the router rule container labels to make the proxy listen to all subdomains (since they are dynamically generated). To do so, I would deploy my app using a Dockerfile in my repository and then update the Container Labels to something like this:

From this : ```traefik.http.routers.http-0-i0s8844ckg8o88ok0w0w0c4g.rule=Host(`app.mydomain.com`) && PathPrefix(`/`)```
To this : ```traefik.http.routers.http-0-i0s8844ckg8o88ok0w0w0c4g.rule=HostRegexp(`app.mydomain.com`, `{subdomain:[a-zA-Z0-9-]+}.app.mydomain.com`) && PathPrefix(`/`)```

I would update the "https" router route as well. This change allow my app to listen to any subdomain instead of only the one configured in the Domains config field (in General section).

Previously, once I hit the Save button, my Container Labels were updated and everything was working as intended after a redeploy.

**[Current version behaviour]**

Today I tried to do the same operation but this time hitting the Save button is now resetting container labels to defaults.

It seems to be an unwanted behaviour of Coolify.
Thanks for your help!

### Steps to Reproduce

1. Create a project, env & resource (any kind, I personally deploy Dockerfiles)
2. Try to edit the Container Labels in Network section
3. Hit Save button
4. Container Labels are back as before, change not applied

### Coolify Version

v4.0.0-beta.375

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Ubuntu 22.04.4 LTS

**Comment by Cinzya** [comment]
Did you enable `Readonly labels` undernath the container labels?
Leaving this unticked, will have the exact effect you just described.

![Image](https://github.com/user-attachments/assets/e18d77ca-1940-4bdb-8adc-03d1f8ffed50)

**Comment by spectaA** [comment]
I completely forgot I ever enabled this feature, thank you so much!
Sorry for the useless issue!

**Comment by gicungdc123** [comment]
hi guys, i got the same error today with v4.0.0-beta.376. i did enable the flag, but after save and reload the config gone.

**Comment by semihsezer** [comment]
I spent 2 hours trying to debug this until I discovered this thread. It is working with v4.0.0-beta.379, but there is a UX issue.

For "Readonly labels" checkmark, it seems that default behavior is `readonly_labels=True `. In that case:
1) Readonly labels should be checked by default
2) You shouldn't be able to edit the custom labels and hit save. Otherwise, the user is confused why it is not saved.

Alternatively, we can rename "Readonly labels" to "Overwrite labels", which matches the observed behavior more accurately.
